.semaphore/semaphore.yml

```diff
@@ -33,11 +34,18 @@ blocks:
       jobs:
         - name: "Install"
           commands:
-            - cache restore
+            - artifact pull project cache.tgz
+            - tar zxvf cache.tgz
+            - rm ~/.m2/settings.xml
             - >
               MAVEN_OPTS="${MAVEN_OPTS} -Xmx3000m" ${MVN} clean install
               -q -ff ${MAVEN_SKIP} ${MAVEN_SKIP_TESTS} -T1C
-            - cache store
+            - tar zcvf cache-post-install.tgz .m2 target
+            - artifact delete workflow cache-post-install.tgz || true
+            - artifact push workflow cache-post-install.tgz
+            - mv cache-post-install.tgz cache.tgz
+            - artifact delete project cache.tgz || true
+            - artifact push project cache.tgz
```

Comment:
maybe we can try without the project level cache for now and see how much of it actually increases build time. All we really care about at project level is maven dependency caching, but this also includes build artifacts so it might be larger than it needs to be and could potentially pollute the build. In the end it might be faster to pull it all from maven CDNs

Reply:
Agreed, and this should be the end state as well.

Reply:
it looks like your PR build [took 5 minutes for the Install step without project cache](https://semaphore.ci.confluent.io/jobs/67cdd810-8c0c-434e-b881-62a279c7b440) compared to [8 minutes for the 24.0.2-confluent branch](https://semaphore.ci.confluent.io/jobs/728860cb-77b2-4f32-9268-b91af1e4248c) , so I think it's safe to remove the project level cache since it doesn't seem to decrease build time.

Reply:
removed and improved.